make.js

```diff
@@ -341,6 +341,7 @@ CLI.test = function(/** @type {{ suite: string; node: string; task: string }} */
             return;
         }
 
+        console.log(nodeVersion)
```

Comment:
```suggestion
        console.log(`Node version: ${nodeVersion}`)
```

Reply:
sorry, accidentally added this line for debugging, already removed it